app/models/lit/source.rb

```diff
@@ -47,8 +47,8 @@ def synchronize
             il.localization_key = LocalizationKey.where(localization_key: il.localization_key_str).first
             unless il.is_duplicate?(r['value'])
               il.save!
-              IncommingLocalization.where(id: il.id).
-                update_all ['translated_value=?', r['value']]
+              IncommingLocalization.where(id: il.id).first.
```

Comment:
This will call ORM and map SQL record to object. In case of synchronization time is crucial, so `'first.update_attributes(translated_value: r['value'])` should be changed into `update_all(translated_value: r['value'])`